client/src/components/PostPage/PostPage.test.js

```diff
@@ -1,9 +1,9 @@
-import React from "react";
-import ReactDOM from "react-dom";
-import PostPage from "./PostPage";
+// import React from "react";
```

Comment:
travis ci cant read the window so it crashes. will have to research how to mock that